### PR TITLE
updated heading Azure Policy or ARM API used for evaluation to Azure Policies or REST APIs used for evaluation

### PR DESCRIPTION
**Pull request by v-rahkuma**
updated heading Azure Policy or ARM API used for evaluation to Azure Policies or REST APIs used for evaluation

### CODE REVIEW

File: Control coverage/Feature/APIManagement.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Use of HTTPS ensures server/service authentication and protects data in transit
         # Refer https://docs.microsoft.com/en-us/powershell/module/az.apimanagement/set-azapimanagementapi
         ```
 
-### Azure Policy or ARM API used for evaluation 
+### Azure Policies or REST APIs used for evaluation
 
 - ARM API to list APIMs and its related property at Subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service?api-version=2019-12-01<br />
```
Comment (jaiswalricha) [inline Control coverage/Feature/APIManagement.md:59]: if you replace ARM in heading why not here?

File: Control coverage/Feature/APIManagement.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ TLS 1.2 is the latest and most secure protocol. Using 3DES Ciphers, TLS protocol
 
     Refer [How do I remediate failing control Azure_APIManagement_DP_Use_Secure_TLS_Version?](https://github.com/azsk/DevOpsKit-docs/blob/master/00c-Addressing-Control-Failures/Readme.md#how-do-i-remediate-failing-control-azure_apimanagement_dp_use_secure_tls_version) to disable the insecure protocols and ciphers using PowerShell.
 
-### Azure Policy or ARM API used for evaluation
+### Azure Policies or REST APIs used for evaluation
 
 - ARM API to list APIMs and its related property at Subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service?api-version=2019-12-01<br />
```
Comment (jaiswalricha) [inline Control coverage/Feature/APIManagement.md:142]: same here

File: Control coverage/Feature/APIManagement.md [hunk 3]
```diff
@@ -222,7 +222,7 @@ Delegation allows you to use your existing website for handling developer sign-i
 
 	 To disable delegation, go to APIM service --> Developer portal --> Delegation --> Uncheck the two check boxes (a) Delegate sign-in & sign-up (b) Delegate product subscription. Read more about delegation here: https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-setup-delegation
 
-### Azure Policy or ARM API used for evaluation 
+### Azure Policies or REST APIs used for evaluation
 
 - ARM API to list APIMs and its related property at Subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service?api-version=2019-12-01 <br />
```
Comment (jaiswalricha) [inline Control coverage/Feature/APIManagement.md:227]: here we are not listing APIm but APIm services

File: Control coverage/Feature/APIManagement.md [hunk 4]
```diff
@@ -258,7 +258,7 @@ If 'validate-jwt' policy is not configured, client can call the API without the
 
 	 For steps to add JWT Validate Token policy please refer: <br>https://docs.microsoft.com/en-us/azure/api-management/api-management-access-restriction-policies#ValidateJWT <br>and <br>https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-protect-backend-with-aad#configure-a-jwt-validation-policy-to-pre-authorize-requests 
 
-### Azure Policy or ARM API used for evaluation 
+### Azure Policies or REST APIs used for evaluation
 
 - ARM API to list APIMs and its related property at Subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service?api-version=2019-12-01<br />
```
Comment (jaiswalricha) [inline Control coverage/Feature/APIManagement.md:263]: same here

File: Control coverage/Feature/APIManagement.md [hunk 5]
```diff
@@ -348,7 +348,7 @@ Enabling OAuth/OpenID connect user authorization ensure that only valid users ha
 
 	 To enable user authorization for an API, go to Azure Portal --> your API management instance --> APIs --> Select API --> Settings -> User Authorization -> Enable 'OAuth 2.0' or 'OpenID connect'. Please refer: https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-oauth2. 
 
-### Azure Policy or ARM API used for evaluation 
+### Azure Policies or REST APIs used for evaluation
 
 - ARM API to list APIMs and its related property at Subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service?api-version=2019-12-01 <br />
```
Comment (jaiswalricha) [inline Control coverage/Feature/APIManagement.md:353]: pls correct on all places

File: Control coverage/Feature/APIManagement.md (hunk 1 above)
Comment (v-rahkuma) [reply Control coverage/Feature/APIManagement.md]: Replaced ARM from sentences too.

File: Control coverage/Feature/APIManagement.md (hunk 3 above)
Comment (v-rahkuma) [reply Control coverage/Feature/APIManagement.md]: Done, changed it to REST API to list APIMs services

File: Control coverage/Feature/APIManagement.md (hunk 5 above)
Comment (v-rahkuma) [reply Control coverage/Feature/APIManagement.md]: Done, corrected on all places

File: Control coverage/Feature/APIManagement.md (hunk 2 above)
Comment (v-rahkuma) [reply Control coverage/Feature/APIManagement.md]: Done

File: Control coverage/Feature/APIManagement.md (hunk 4 above)
Comment (v-rahkuma) [reply Control coverage/Feature/APIManagement.md]: Done

**Review by jaiswalricha — APPROVED** [review]

**Review by Aboli-msft — APPROVED** [review]

**Pull request merged**